src/sprite/serialize.rs

```diff
@@ -2,7 +2,7 @@ use resvg::usvg::Rect;
 use serde::ser::SerializeSeq;
 use serde::{Serialize, Serializer};
 
-/// Custom Serde field serialiser for [`Rect`].
+/// Custom Serde field serializer for [`Rect`].
```

Comment:
```suggestion
/// Custom Serde field serialiser for [`Rect`].
```

I _think_ I'm at the fifth stage of grief and I've accepted that I need to write code in US English, but prose is a different matter. Serde's `Serializer` is a type of serialiser, and so the One True English (aka British English) should be used here ;)

Reply:
lol, i didn't even realize the british eng spelled it differently. TIL